pkg/resource/scalable_target/testdata/scalable_target/sdkapi/describe_success.yaml

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
all of the files in `sdkapi` folder should be `.json` not `.yaml`

Reply:
good catch, done.